fineract-provider/src/main/java/org/apache/fineract/organisation/workingdays/domain/WorkingDays.java

```diff
@@ -74,6 +78,8 @@ public Boolean getExtendTermForDailyRepayments(){
         return this.extendTermForDailyRepayments;
     }
 
+    public Boolean getExtendTermForRepaymentsOnHolidays() { return this.extendTermForRepaymentsOnHolidays; }
+
     public Map<String, Object> update(final JsonCommand command) {
```

Comment:
I apologize that I didn't mention the fact that I was asking for feedback on how I added the changes. That said, I will add in the rest of the backend changes in order to make this an acceptable PR when I get a little extra time.